insights/tests/parsers/test_mount.py

```diff
@@ -61,6 +61,7 @@
 PROC_MOUNT_QUOTES = """
 /dev/mapper/rootvg-rootlv / ext4 rw,relatime,barrier=1,data=ordered 0 0
 tmpfs /var/lib/containers/storage/overlay-containers/ff7e79fc09c/userdata/shm tmpfs rw,nosuid,nodev,noexec,relatime,context="system_u:object_r:container_file_t:s0:c184,c371",size=64000k 0 0
+tmpfs /var/lib/containers/storage/overlay-containers/aa7e79fc09c/userdata/shm tmpfs rw,nosuid,nodev,noexec,relatime,context="system_u:object_r:container_file_t:s0",size=64000k 0 0
 """
```

Comment:
Hi @bfahr ,  
Seeing this adding  test case `...,context="system_u:object_r:container_file_t:s0",size=64000k`, I guess that maybe we can also be compatible for a mount option case like: 
`rw,nosuid,nodev,noexec,relatime,context="system_u:object_r:container_file_t:s0:c184,some_inside_quote_option,c371",size=64000k` 
What do you think ?

Reply:
I think @JoySnow might mean: if we can add a test like `rw...,context="system_u:object_r:container_file_t:s0:c184,c371,c372",size=...`

Although the `context` was parsed correctly:
```python
assert device.mount_options.context == "system_u:object_r:container_file_t:s0:c184,c371,c372"
# it's True
```
However, the second `c371` was also parsed individually in the `else` part as a Boolean option
```python
assert  hasattr(device.mount_options, "c371") is True
assert device.mount_options.c371 is True
These are also `True`, but it should be not (right?)

```

Reply:
Good catch @JoySnow!  I've added that test case and fixed the problem.